src/pages/Area51Page.tsx

```diff
@@ -0,0 +1,52 @@
+import "./Area51Page.scss";
+import imageSecret from "../assets/image-martian-selfie.jpeg";
```

Comment:
When you move the image to `/public`, this will simplify to
```suggestion
import imageSecret from "/image-martian-selfie.jpeg";
```